docs-ui/components/colorChip.tsx

```diff
@@ -20,6 +20,25 @@ type Props = {
   theme: Theme;
 };
 
+type OuterWrapProps = {
+  large: boolean;
+};
+
+type WrapperProps = {
+  large: boolean;
```

Comment:
Would a `size` prop be more appropriate as like `size="md"` or `size="lg"`

Reply:
Considered that but thought that since there are only two sizes, a boolean might work better. Could just use `size` instead if it doesn't make too much sense for you.

Reply:
The reason I might suggest size, is in the case where we add more. Just gives that extra bit of flexability